### DISCUSSION TITLE
Statistician - Need Data frame of Broadway shows

### DISCUSSION THREAD

**Discussion by samipierre** [pair-cg/general-contributions#49] [User Stories]
# 1. Need:
I want to estimate the effect of Tony award wins and nominations on Broadway shows from 2011-2019 run length and capacity sold on a weekly basis . 
I am interested only in the show that were eligible for any Tony award nomination/win for the 2011-2019 Tony awards in all categories
 
# 2. Action:
- Gathering weekly show data 
   - for example from broadway.com
   - the start date of each week the show ran (North American)
   - the year the show was eligible for the awards
   - weekly show capacity 
   - length of a shows run in number of weeks
- gathering Tony data from Tony Awards website (
   - for each award, what shows were nominated and which show won
   - dates of each Tony awards
   - when the nominations were announced 
   - when the cut-off date for eligibility was
- create concordance document to match show names to correct shows 

# 3. Desired Outcome:
To estimate the effect that a win or nomination of each Tony award has on a shows run length and weekly capacity (measures for profitability)

# 4. Perspective:
researcher

# 5. Stakeholder Group:
researchers
producers
archivists
Broadway Data organizations (Broadway.com, the Broadway League)

# 6. Additional info:
_Here's more information._

**Comment by drosu** [comment]
What is the purpose of your research

**Reply by drosu** [reply]
What is the time frame for the research question?

**Reply by drosu** [reply]
Are you considering all Tony award categories?

**Reply by drosu** [reply]
what kind of data do you need?

**Reply by drosu** [reply]
Isn't the desired outcome producing an estimation of the effect Tony award wins and nominations have on the profitability of the Broadway shows?
